### ISSUE TITLE
Cmd-Click navigation links should open page in new tab without affecting current tab

### ISSUE THREAD

**Issue by joeltg** [pubpub/pubpub#1772]
_Before filing your bug report, please search existing issues to make sure a similar one doesn't already exist. If one does, please leave a comment or reaction in support of the issue rather than creating a new one._

_To fill out the report, please leave the below headers (**) in tact, and fill out each section. If a section isn't applicable, leave it blank or write n/a under it. Please do not delete any template sections._

### What went wrong, step-by-step?
1. Cmd-Click Create Pub just navigates to a new pub draft in the current page (Firefox and Safari)
2. Cmd-Click View Profile navigates to the profile page in the current tab and *also* opens the profile page a new tab (Firefox only (?))
3. Cmd-Click Privacy settings has the same behavior as View Profile (Firefox only (?))
...

### What did you expect to happen?

In all of these cases I'd expect the link to open in a new tab without affecting the current tab. This is probably due to some JS click handler / client side routing thing.

### What browser/operating system were you using?

Firefox and Safari

===

**Comment by idreyn** [comment]
> This is probably due to some JS click handler / client side routing thing.

Yep, behind the scenes it's POSTing `/api/pubs` and then setting `window.location.href` to the Pub we get back. I plan to add a new, non-API route `/pub/new` (and `/pub/new/:from-template-slug`) that would let us implement the New Pub button as a real anchor element.
